doc/source/configure.rst

```diff
@@ -25,61 +17,93 @@ Configuration examples assume there are three |PXC| nodes:
 | Node 3 | pxc3      | 192.168.70.63 |
 +--------+-----------+---------------+
 
-If you are running Debian or Ubuntu,
-add the following configuration variables to :file:`/etc/mysql/my.cnf`
-on the first node::
+1. Stop the |PXC| server:
+
+   .. code-block:: bash
+
+      $ sudo service mysql stop
+
```

Comment:
For 8.0 service isn't started after installation, so this step isn't needed anymore (unless user manually started the service after installation)